policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/basic_personal_amount/additional_amount/income_threshold.yaml

```diff
@@ -0,0 +1,12 @@
+description: Nova Scotia lower income threshold for maximum basic personal amount in basic personal amount.
```

Comment:
rephrase like other descriptions - though is this parameter needed? It's not in the formula

Reply:
This was not resolved